README.md

```diff
@@ -1183,6 +1184,181 @@ to compute the checksum, such as `MD5` or `SHA-1`.
 }
 ```
 
+#### Custom Tika Parsers
+
+It might occur that one or more existing Tika parsers do not provide the intended information, or just do not exist.
+This setting allows to use a custom parser instead.
+The parsers must be provided as a .jar, but does not need to be on any classpath.
+Note that this is an array. Here an example for just one
+
+```json
+{
+  "name": "test",
+  "fs": {
+    "custom_tika_parsers": [
+      {
+        "class_name": "org.me.MyParser",
+        "path_to_jar": "/some/full/path/to/myParser-0.0.1-SNAPSHOT.jar",
+        "mime_types": ["application/dns", "or-another-mimetype-from-tika"]
+      }
+    ]
+  }
+}
+```
+
+Some info about creating a custom parser is available [here](https://tika.apache.org/1.17/parser_guide.html)
+Or use a existing parser as a blueprint. Make sure to choose the correct branch.
+At the time of this writing fscrawler uses Tika 1.17, while on github the main branch is 2.x.
+The parsers from ["branch_1x"](https://github.com/apache/tika/tree/branch_1x/tika-parsers/src/main/java/org/apache/tika/parser) should work fine.
+
+To build the custom parser separately, a pom file can be derived from the tika-parsers [pom.xml](https://github.com/apache/tika/blob/branch_1x/tika-parsers/pom.xml).
+Probably a lot can be left out. Here is an example which required fontbox (guess still to long, but worked).
```

Comment:
WDYM with "guess still to long, but worked"

too long? Is it a comment you are making here? If so I believe this is not needed.

Reply:
Just to indicate that I didn’t really evaluated the Pom.xml, but stopped when it build without errors.
Should get marked someway as to-do

Reply:
Changed it to
(The exclusions are copied 1:1 from fscrawler's pom.xml, to be on the safe side)